code/modules/client/preferences.dm

```diff
@@ -1552,6 +1557,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					features["mcolor"] = pref_species.default_color
 				user << browse(null, "window=speciespick")
 				ShowChoices(user)
+				all_quirks = list()
```

Comment:
Couldn't you just iterate through all the quirks and check if there's any incompatibilities? This seems extremely inconvenient.

Or just let them select it and not give it to them on spawn when they're playing an incompatible species? I don't know, anything but this

Reply:
This is the thing that was being very naughty and not working and I needed breakpoints for, which were not working and currently I'm busy, it's a very inconvenient placeholder that is only still there because I couldn't change it yet

Reply:
Makes sense, just remember to fix it when you can.